windows.md

```diff
@@ -554,23 +606,23 @@ With those compatibility things out of the way, you're ready to start the system
     ```
     This step will save your GitHub password so that you don't need to enter it every time.<br><br> -->
 
-25. Copy the following text, paste it in Hyper and hit return.<br><br>
+28. Copy the following text, paste it in Hyper and hit return.<br><br>
     ```bash
     git config --global init.defaultBranch main
     ```
     This step will change the default Git branch from `master` to `main` (see https://github.com/github/renaming).<br><br>
-26. Copy and run each of these lines separately in Hyper.<br><br>
+29. Copy and run each of these lines separately in Hyper.<br><br>
     ```bash
     git config --global core.autocrlf false
     git config --global core.eol lf
     ```
     This step will improve line breaks compatibility on Windows.
-27. Go back to GitHub, and go to your profile page by clicking on your avatar at the top right and selecting **Your profile**<br><br>
+30. Go back to GitHub, and go to your profile page by clicking on your avatar at the top right and selecting **Your profile**<br><br>
     <img src="./general-github-your-profile.png"><br><br>
     Copy the `github.com/...` URL in the address bar of your browser, for use in the next step.
-28. Open the Start menu and start Slack. Log in to the UpLeveled Slack. Send your GitHub profile URL to [Lukas](https://upleveled.slack.com/team/U026UPA0TKL). Also send your Google Account email address to Lukas (if you haven't already).
-29. <a name="specs"></a>Open the start menu, type "Settings" and open the app (or click on the cog on the left). Select "System" and "About". Under "Device specifications", click the Copy button and paste this to Lukas. Under "Windows specifications", click the Copy button and paste this to Lukas.
-30. On your phone, go to the app store and install Slack on your phone. Log in to the UpLeveled Slack.
+31. Open the Start menu and start Slack. Log in to the UpLeveled Slack. Send your GitHub profile URL to [Lukas](https://upleveled.slack.com/team/U026UPA0TKL). Also send your Google Account email address to Lukas (if you haven't already).
+32. <a name="specs"></a>Open the start menu, type "Settings" and open the app (or click on the cog on the left). Select "System" and "About". Under "Device specifications", click the Copy button and paste this to Lukas. Under "Windows specifications", click the Copy button and paste this to Lukas.
+33. On your phone, go to the app store and install Slack on your phone. Log in to the UpLeveled Slack.
```

Comment:
seems like these changes are unrelated?

if the changes here are about the numbering, we can pull this out into a separate PR and merge it quickly

Reply:
New PR
https://github.com/upleveled/system-setup/pull/66